test/index.js

```diff
@@ -2,6 +2,7 @@
 
 'use strict';
 
+const assert = require('assert');
```

Comment:
Any reason to use `assert` over chai `expect`?

Reply:
Fixed!